docs/source/api/xskillscore.Contingency.accuracy.rst

```diff
@@ -1,6 +1,6 @@
-xskillscore.Contingency.accuracy
+﻿xskillscore.Contingency.accuracy
```

Comment:
can we somehow stop this? some pre-commit thing I guess.

Reply:
I'll auto rerun the api gen and it might fix itself. I thought it added all of these changes since I updated intersphinx but it looks weirdly like it's just updating the name in there.